0.data-download/scripts/data_loader.py

```diff
@@ -43,21 +43,22 @@ def load_data(data_directory, adult_or_pediatric = "all"):
        sample_df = sample_df.query("age_categories == @adult_or_pediatric").reset_index(drop=True)
        samples_to_keep = sample_df.reset_index(drop=True).DepMap_ID.tolist()  
        dependency_df = dependency_df.query("DepMap_ID == @samples_to_keep").reset_index(drop=True)
-       
     
     return sample_df, dependency_df
+    
+    sample_df = sample_df.set_index("DepMap_ID").reset_index()
 
 
 
 def load_train_test_data(data_directory, train_or_test = "all"):
     # define directory paths
     training_data_file = pathlib.Path(data_directory, "VAE_train_df.csv")
     testing_data_file = pathlib.Path(data_directory, "VAE_test_df.csv")
-    gene_statistics_file = pathlib.Path(data_directory, "genes_variances_and_t-tests_df.csv")
+    gene_statistics_file = pathlib.Path(data_directory, "genes_variances_and_t-tests_df.csv")    
     
     train_df = pd.read_csv(training_data_file)
     test_df = pd.read_csv(testing_data_file)
-    gene_stats = pd.read_csv(gene_statistics_file)
+    gene_stats = pd.read_csv(gene_statistics_file)   
```

Comment:
hmm, looks like there are some weird empty spaces being added to this file.

[`black`](https://github.com/psf/black) will take care of this automagically